examples/with-remix/app/root.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import type { MetaFunction } from '@remix-run/node';
+import { Links, LiveReload, Meta, Outlet, Scripts, ScrollRestoration } from '@remix-run/react';
+import styles from 'rsuite/dist/rsuite.css';
+
+export const meta: MetaFunction = () => ({
+  charset: 'utf-8',
+  title: 'New Remix App With RSuite',
+  viewport: 'width=device-width,initial-scale=1'
+});
+
+export default function App() {
+  return (
+    <html lang="en">
+      <head>
+        <Meta />
+        <Links />
+      </head>
+      <body>
```

Comment:
It is best to add a `padding` to the body.
![image](https://user-images.githubusercontent.com/1203827/221543773-fce27f4f-12e3-4916-8088-38c0e54cb74c.png)

Reply:
body padding via shared style